pkg/api/server.go

```diff
@@ -806,23 +875,57 @@ func (h handler) getAPIsClusterResources(w http.ResponseWriter, r *http.Request)
 		})
 		filenames = []string{filepath.Join(h.clusterData.ClusterResourcesDir, fmt.Sprintf("%s.json", sbctlutil.GetSBCompatibleResourceName(resource)))}
 		if err != nil {
-			log.Println("failed to get cluster-role-binding files from dir", err)
+			log.Error("failed to get cluster-role-binding files from dir: ", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+	case "selfsubjectaccessreviews":
+		body, err := io.ReadAll(r.Body)
+		if err != nil {
+			log.Error("failed to read request body: ", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		decoded, gvk, err := sbctl.Decode(resource, body)
+		if err != nil {
+			log.Error("failed to decode wrapped ", resource, ": ", err)
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+		if selfReview, ok := decoded.(*authorizationv1.SelfSubjectAccessReview); ok {
+			selfReview.Status.Allowed = true // In sbctl, we always allow self access reviews
+			JSON(w, http.StatusOK, selfReview)
+		} else {
+			log.Warnf("We do not know gvk: %s\n", gvk)
+			JSON(w, http.StatusNotFound, errorNotFound)
+		}
+		return
 	}
 
 	for _, fileName := range filenames {
+		// If we know the file does not exist, just respond with an empty list
+		if !fileExists(fileName) {
+			obj := unstructured.UnstructuredList{}
+			obj.SetGroupVersionKind(schema.GroupVersionKind{
+				Group:   mux.Vars(r)["group"],
+				Version: mux.Vars(r)["version"],
+				Kind:    resource,
+			})
+
+			result = &obj
+			break
```

Comment:
Do nothing and continue. Set result to empty object if no data was found in the entire loop.